commands/channel.go

```diff
@@ -88,11 +88,12 @@ Channel can be specified by [team]:[channel]. ie. myteam:mychannel or by channel
 }
 
 var RestoreChannelsCmd = &cobra.Command{
-	Use:   "restore [channels]",
-	Short: "Restore some channels",
+	Use:     "restore [channels]",
```

Comment:
As we plan to deprecate `restore`, instead of creating an alias, we can mark this command as deprecated and create an `unarchive` command that will be the preferred path to follow, like we've done in 13d08b4ecbf4573e405cf3eadb805032278e1d40

Reply:
Oh, didn't think that we deprecate it right away :) Will update accordingly 👍

Reply:
Adding the `Deprecated` prop still allows you to use it, it just prints a warning before running, and does not show the command in the docs or suggestions. I'd prefer it to say something like "command is *marked* as deprecated", but it works nonetheless :)

Reply:
It's still a good thing to have, `cobra` continues to surprise me 😄